### ISSUE TITLE
Cannot find button 

### ISSUE THREAD

**Issue by nattzp** [jonkpirateboy/Pass-fur-alle#52]
When setting firstDate to true, I get this error message after ctrl+c: Unable to locate element: //*[contains(@aria-label,"202")]
And when inspecting the webpage I can't seem to find an element with "202"...
Does anyone else have this problem?

**Comment by jonkpirateboy** [comment]
This is now closed since Pass für alle has changed from Python to Javascript
